### ISSUE TITLE
Boxes problem when more than 2 columns

### ISSUE THREAD

**Issue by signo** [scribu/wp-scb-framework#9]
When i define a 3 column setup the boxes are sized right but they can only be positioned on the first 2 columns, the third column is not available to position a box,

When i define a 4 column setup they are sized accordingly but only the first 3 columns can be used.

This was tested in win7, Firefox  12 and Chrome 18. WordPress 3.3.1.

The code used:

```php
<?php
class Signo_Boxes_Page extends scbBoxesPage {

	function setup() {
		$this->args = array(
			'page_title' => 'Signo Cajas',
			'columns' => 4,
		);

		$this->boxes = array(
			array( 'settings', 'Caja de Opciones', 'normal' ),
			array( 'center1', 'Opciones al Centro', 'side' ),
			array( 'center2', 'Opciones al Centro', 'column3' ),
			array( 'right', 'Opciones a la Derecha', 'column4' ),
		);
	}

	function settings_box() {
		echo html( 'h2', 'opciones de Signo Creativo' );

	}
	function center1_box() {
		echo html( 'p', 'Esta es la caja en el centro.' );
	}

	function center2_box() {
		echo html( 'p', 'Esta es la caja en el centro.' );
	}

	function right_box() {
		echo html( 'p', 'Esta es la caja a la derecha.' );
	}
}
```

**Comment by LoreleiAurora** [comment]
#43 should fix this problem :)

**Comment by scribu** [comment]
Merged #43
